src/platform/forms/save-in-progress/RoutedSavablePage.jsx

```diff
@@ -1,22 +1,19 @@
+import { setData, uploadFile } from 'platform/forms-system/src/js/actions';
+import { FormPage } from 'platform/forms-system/src/js/containers/FormPage';
```

Comment:
These imports shouldn't be at the top. I think imports from `node_modules` should always be first, then platform, then other imports... I don't know if that's the preferred order, but again, this is a personal preference.

Reply:
I think not importing React first in some places causes problems but this isn't enforced by eslint. 

In general you shouldn't just stage all of your changes when you make a commit. I think it's better to do something like `git add -p` and step through your changes. That way you only add the stuff to the commit that actually matters for your change.